.github/workflows/dev-workflow.yml

```
@@ -191,32 +191,57 @@ jobs:
+  #     - uses: cargo-bins/cargo-binstall@main
 
-      - name: Setup pnpm
-        uses: pnpm/action-setup@v2
-        with:
-          version: 10
-          run_install: true
+  #     - name: Install dioxus-cli
+  #       run: |
+  #         cargo binstall dioxus-cli --force
+  #         cargo binstall toml-cli --force
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)